content/docs/beyond-the-basics/public-attestation/index.md

```diff
@@ -1,14 +1,175 @@
 ---
- title: "Public Assets vs Private Assets"
- description: "Transparency through Public Attestation"
+ title: "Public Attestation"
+ description: "Public Assets vs Private Assets"
```

Comment:
This should now be ` vs permissioned assets`?

Reply:
Yep, thanks for catching!